src/Sulu/Component/Content/Compat/Metadata.php

```diff
@@ -14,7 +14,7 @@
 use JMS\Serializer\Annotation\Type;
 
 /**
- * holds localized metadata.
+ * @internal this class can be removed at any time and you should not build on top of it.
  */
 class Metadata
```

Comment:
Need to have a deeper look how to get rid of this or we move it to the AdminBundle metadata classes. Currently mostly fails during parsing webspace loading as NavigationContext using this class internal for get translated item.